airflow/providers/openlineage/provider.yaml

```diff
@@ -104,6 +104,13 @@ config:
         example: full.path.to.ExtractorClass;full.path.to.AnotherExtractorClass
         default: ~
         version_added: ~
+      custom_run_facets:
+        description: |
+          Register custom run facet functions by passing a string of semicolon separated full import paths.
+        type: string
+        example: full.path.to.custom_facet_function;full.path.to.another_custom_facet_function
+        default: ''
+        version_added: 1.8.0
```

Comment:
This will probably be 1.10.0 as 1.9.0 should be released before this gets merged.

Reply:
Addressed in c1bc84efc2a454dc5eb4c0e33eb206506b06e445.